packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -475,6 +596,143 @@ describe('meeting', () => {
       });
     });
 
+    describe('getMeetingDetailsVerbose', () => {
+      const allowedContexts = [
+        FrameContexts.sidePanel,
+        FrameContexts.meetingStage,
+        FrameContexts.settings,
+        FrameContexts.content,
+      ];
+
+      Object.values(FrameContexts).forEach((context) => {
+        if (allowedContexts.some((allowedContext) => allowedContext === context)) {
+          it(`FRAMELESS: should successfully get the verbose meeting details. context: ${context}`, async () => {
+            await utils.initializeWithContext(context);
+
+            const promise = meeting.getMeetingDetailsVerbose();
+
+            const message = utils.findMessageByFunc('meeting.getMeetingDetails');
+            expect(message).not.toBeNull();
+            expect(message?.args?.length).toBe(1);
+
+            const shouldGetVerboseDetails = true;
+            expect(message?.args?.[0]).toEqual(shouldGetVerboseDetails);
+
+            const callbackId = message?.id;
+            const details: meeting.IMeetingDetails | meeting.ICallDetails = {
+              scheduledStartTime: '2020-12-21T21:30:00+00:00',
+              joinUrl:
+                'https://teams.microsoft.com/l/meetup-join/19%3ameeting_qwertyuiop[phgfdsasdfghjkjbvcxcvbnmyt1234567890!@#$%^&*(%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%226b33ac33-85ae-4995-be29-1d38a77aa8e3%22%7d',
```

Comment:
Do we need this url to be of this format for the unit test?

Reply:
I don't think the format is too important, because we're just checking for equality between the test data and the returned data. So, I just reused this link from a previous test case.